src/userbase-js/src/db.js

```diff
@@ -524,6 +524,7 @@ const insertItem = async (params) => {
       case 'ItemIdCannotBeBlank':
       case 'ItemIdTooLong':
       case 'ItemMissing':
+      case 'ItemCannotBeUndefined':
```

Comment:
Why not use the ItemMissing error?

Reply:
I was debating it -- it felt nicer to a developer to just be explicit.

Right now, ItemMissing is for when there literally is no `item` parameter provided to the function, which may happen for example if the user has a typo or forgets to include it.

But if the developer explicitly passes the item parameter set to undefined, then the developer gets the super obvious ItemCannotBeUndefined error.

I don't have a strong opinion, if you feel it would be better to keep it slim and just use ItemMissing for both cases above, happy to do that. I think a developer would probably very easily understand it can't be undefined based on ItemMissing too. So happy to do it.

Reply:
Another thing I was thinking:

I felt like it would be a little odd that setting the item to undefined returns ItemMissing but setting the item to null is allowed. ItemCannotBeUndefined tells the developer that undefined is something explicitly not allowed. The developer might be more inclined to try setting it to null instead after receiving ItemCannotBeUndefined versus ItemMissing.

Super minor details to get hung up on. I’m cool with whatever

Reply:
Update: went with the general catch all of `ItemInvalid` for items that cannot be serialized to JSON via JSON.stringify. This includes undefined items